### ISSUE TITLE
Limit Sphinx version to 1.6.7

### ISSUE THREAD

**Issue by Glutexo** [RedHatInsights/insights-core#1385]
The [Sphinx](https://pypi.org/project/Sphinx/) package currently [required](https://github.com/RedHatInsights/insights-core/blob/1889616afc90bef003f9195ee142d2de98d72b5b/setup.py#L51) does not support Python 2.6 in its most recent version [1.7.7](https://pypi.org/project/Sphinx/1.7.7/). A little older version [1.6.7](https://pypi.org/project/Sphinx/1.6.7/) still supports it, but there are still come conflicts when installing: [_entrypoints_](https://pypi.org/project/entrypoints/) required by this version can’t install.

This will require further inspection on how to fix the dependencies so the packages install without any conflicts. It is also necessary to check that docs build fine with this version:

```
sphinx-build -W -b html -qa -E docs docs/_build/html
```

Related to #1373 and #1261 / #1369.

**Comment by Glutexo** [comment]
It looks like there is no way to support documentation on Python 2.6. [_entrypoints_](
https://pypi.org/project/entrypoints/) does not support it at all and so doesn’t [_nbconvert_](https://pypi.org/project/nbconvert/). These are required by [_nbsphinx_](https://pypi.org/project/nbsphinx/0.2.0/) [listed](https://github.com/RedHatInsights/insights-core/blob/1889616afc90bef003f9195ee142d2de98d72b5b/setup.py#L52) in [_setup.py_](https://github.com/RedHatInsights/insights-core/blob/1889616afc90bef003f9195ee142d2de98d72b5b/setup.py).

Fortunately enough, this is not necessary neither for runtime nor for development. Once #1369 is merged, _Sphinx_ won‘t interfere neither with `pip install -e .[develop]` nor with `pip install -e .[testing]`. That will allow to use virtual environments for #1342 and for a follow-up PR to #1369, which doesn’t exist yet.
